templates/workflow/workflow.tpl

```diff
@@ -16,22 +15,21 @@
 {assign var=submissionTitleSafe value=$submission->getLocalizedTitle()|strip_unsafe_html}
 {assign var="pageTitleTranslated" value=$primaryAuthor->getLastName()|concat:", ":$submissionTitleSafe}
 {include file="common/header.tpl" suppressPageTitle=true}
-{/strip}
 
 <script type="text/javascript">
 	// Initialise JS handler.
 	$(function() {ldelim}
 		$('#submissionWorkflow').pkpHandler(
-			'$.pkp.pages.workflow.WorkflowHandler'
+			'$.pkp.pages.workflow.WorkflowHandler',
 		);
 	{rdelim});
 </script>
 
-<div id="submissionWorkflow">
+<div id="submissionWorkflow" class="pkp_submission_workflow">
 
-{url|assign:submissionHeaderUrl op="submissionHeader" submissionId=$submission->getId() stageId=$stageId contextId="submission" escape=false}
-{load_url_in_div id="submissionHeaderDiv" url=$submissionHeaderUrl class="pkp_page_header"}
+	{url|assign:submissionHeaderUrl op="submissionHeader" submissionId=$submission->getId() stageId=$stageId contextId="submission" escape=false}
+	{load_url_in_div id="submissionHeaderDiv" url=$submissionHeaderUrl class="pkp_page_header"}
 
-{include file="controllers/notification/inPlaceNotification.tpl" notificationId="workflowNotification" requestOptions=$workflowNotificationRequestOptions}
```

Comment:
sure, after he got it back here https://github.com/NateWr/pkp-lib/commit/1663c35df13e833b05e8c954e07b6f593f5a6c29 :)